code/modules/antagonists/nukeop/equipment/nuclear_challenge.dm

```diff
@@ -70,6 +69,11 @@ GLOBAL_LIST_EMPTY(jam_on_wardec)
 		to_chat(usr, span_warning("Invalid war declaration."))
 		return
 
+	for(var/obj/item/circuitboard/computer/syndicate_shuttle/board as anything in GLOB.syndicate_shuttle_boards)
+		if(board.challenge)
+			to_chat(usr, span_warning("War has been delcared already!"))
```

Comment:
```suggestion
			tgui_alert(usr, "War has already been declared!", "Can't Declare!")
```

spellcheck + formatting fix- also it should really be a tgui_alert() instead because everything else in this proc uses a tgui_alert

also this should be moved above the whole "write out a custom war declaration message" stuff because some admin is going to spend like three minutes typing out something very good only to be disappointed when they find out that someone already declared war somehow

Reply:
Moving it would be redundant, we have to check before war is declared regardless so like the admin is owned in both situations

Reply:
womp